packages/access-client/src/agent.js

```diff
@@ -27,7 +27,13 @@ export { AgentData, Access, Space, Delegation, Schema }
 export * from './agent-use-cases.js'
 
 const HOST = 'https://up.storacha.network'
-const PRINCIPAL = DID.parse('did:web:storacha.network')
+const PRINCIPAL = DID.parse('did:web:up.storacha.network')
```

Comment:
why did this change? will this have any upredictable effects?

Reply:
also this gets overridden (https://github.com/storacha/upload-service/blob/main/packages/access-client/src/agent.js#L67) when connecting to staging and dev environments, will that be a problem for the code that uses IDENTITIES below?